src/components/oauthDocs/BuildingOIDCContent.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { HashLink } from 'react-router-hash-link';
+import { SectionHeaderWrapper } from '../index';
+import { onHashAnchorClick } from '../../utils/clickHandlers';
+import { AuthCodeFlowContent } from './AuthCodeFlowContent';
+import { PKCEAuthContent } from './PKCEAuthContent';
+
+const BuildingOIDCContent = (): JSX.Element => (
+  <section aria-labelledby="Building OpenID Connect Applications">
+    <SectionHeaderWrapper heading="Building OpenID Connect Applications" id="building-openid" />
```

Comment:
since these anchors are visible to the user, i think including "connect" or the "oidc" acronym would probably be beneficial here.

```suggestion
    <SectionHeaderWrapper heading="Building OpenID Connect Applications" id="building-oidc-apps" />
```

Reply:
done